src/pavan/Assignment17/DiffrenceOfSum.java

```diff
@@ -0,0 +1,44 @@
+/*
+ Program 2: return a difference between sum of even number and odd numbers.	
+	From given numbers, count the odd and even numbers.
+	input : 10,23,26,27,88,87,65 
+ */
+
+package pavan.Assignment17;
+
+public class DiffrenceOfSum {
+	int evencount=0;
+	int additionofeven=0;
+	int oddcount=0;
+	int additionofodd=0;
+	void countNumbers(int[]args) {
+		for (int index=0; index<7;index++) {
+			if (args[index]%2==0){
+				evencount++;             
+				additionofeven = args[index]+additionofeven;	
+				
+
+			}
+		}//System.out.println(additionofeven);
+		
+		for (int index=0; index<7; index++) {
```

Comment:
why do you require two loop to identify odd and even?
you can handle in single loop just adding if else condition 

for (int index=0; index<7;index++) {
			if (args[index]%2==0){
				evencount++;             
				additionofeven = args[index]+additionofeven;	
                         }else {
                                 oddcount++;
				additionofodd= args[index]+additionofeven;
		}